podcasts/PodcastListViewController.swift

```diff
@@ -177,6 +177,17 @@ class PodcastListViewController: PCViewController, UIGestureRecognizerDelegate,
                     self.setupBannerAd(promotion: promotion, shouldAnimate: false)
                 }
             }
+        } else {
+            if bannerAdModel != nil {
```

Comment:
The only thing I wonder if whether `bannerTask?.cancel()` or it should be moved outside of the `if FeatureFlag.bannerAdPodcasts.enabled && !SubscriptionHelper.hasActiveSubscription() {` block.

I'm not sure it really matters much but I could see a theoretical case where the task is running and flag or subscription changes and then the task comes back.